libr/anal/esil.c

```diff
@@ -4,6 +4,7 @@
 #include <r_types.h>
 #include <r_util.h>
 #include <r_db.h>
+#include <r_endian.h>
```

Comment:
can't this be included into r_types.h ? i dont think we need to add a new include everywhere just for this basic thing